api/dsService/dsUtil.js

```diff
@@ -1,23 +1,59 @@
-const axios = require('axios')
-
-// model imports
-const incidentsModel = require('../incidents/incidentsModel')
+const axios = require('axios');
+const knex = require('../../data/db-config');
 
 const dsURL = process.env.DS_API_URL;
 
 module.exports = {
-    dsFetch
-}
+  dsFetch,
+};
 
 function dsFetch() {
-    return axios
+  const incomingIncidents = [];
+
+  return axios
     .get(`${dsURL}/getdata/`)
     .then((response) => {
-      response.data.forEach((element) => {
-        incidentsModel.createIncident(element);
+      response.data.forEach((incident) => {
+        let newIncident = {
+          incident_id: incident.case_id,
+          src: JSON.stringify(incident.links),
+          categories: JSON.stringify(incident.tags),
+          city: incident.city,
+          state: incident.state,
+          title: incident.title,
+          lat: incident.lat,
+          long: incident.long,
+          desc: incident.description,
+          date: incident.dates,
+          verbalization: incident.verbalization,
+          empty_hand_soft: incident.empty_hand_soft,
+          empty_hand_hard: incident.empty_hand_hard,
+          less_lethal_methods: incident.less_lethal_methods,
+          lethal_force: incident.lethal_force,
+          uncategorized: incident.uncategorized,
+        };
+        
```

Comment:
Looks like there are a few formatting test errors in this file...everything else looks good!